plugin/src/functionalTest/kotlin/io/qalipsis/gradle/cloud/CloudRunQalipsisTaskFunctionalTest.kt

```diff
@@ -0,0 +1,655 @@
+                        campaignName.set("Dummy campaign configuration")
+                        speedFactor.set(2.0)
+                        startOffsetMs.set(2000)
+                        campaignTimeout.set("PT1H30M")
+                        hardTimeout.set(false)
+                        scenario("My scenario 1") {
+                            minionsCount = 7
+                            zones {
+                               "CH" to 45
+                               "AS" to 55
+                            }
+                            profile {
+                                more(
+                                    periodMs = 11, 
+                                    minionsCountProLaunchAtStart = 23, 
+                                    multiplier = 2.0, 
+                                    maxMinionsCountProLaunch = 7
+                                )
+                            }
+                        }
+                    }
+                }
+            """.trimIndent()
+        )
+        mockServerClient.`when`(
+            request()
+                .withPath("/api/campaigns")
+                .withMethod("POST")
+                .withHeader(Header("Authorization", "Bearer my-token"))
+                .withBody(
+                    JsonBody.json(
+                        """
+                            {
+                            "name": "Dummy campaign configuration",
+                            "speedFactor": 2.0,
+                            "startOffsetMs": 2000,
+                            "timeout": "PT1H30M",
+                            "hardTimeout": false,
+                            "scenarios": {
+                                "My scenario 1": {
+                                    "minionsCount": 7,
+                                    "zones": {
+                                        "CH": 45,
+                                        "AS": 55
+                                    },
+                                    "executionProfile": {
+                                        "profile": "PROGRESSING_VOLUME",
+                                        "periodMs": 11,
+                                        "minionsCountProLaunchAtStart": 23,
+                                        "multiplier": 2.0,
+                                        "maxMinionsCountProLaunch": 7
+                                    }
+                                }
+                            }
+                        }
+                        """.trimIndent()
+                    )
+                )
+        ).respond(
+            response()
+                .withStatusCode(200)
+                .withHeader(Header("Content-Type", "application/json; charset=utf-8"))
+                .withBody(
+                    """
+                        {
+                            "version": "2025-04-28T13:09:43.533Z",
+                            "key": "Dummy campaign configuration",
```

Comment:
In reality, the key is a random unique value, not the name. Please use such a value, for example: `r0g0u1piyI`